watchtower/__init__.py

```diff
@@ -45,6 +45,20 @@ def _boto_filter(record):
     return True
 
 
+def _size(msg):
```

Comment:
Can you move _size to be a method of CloudWatchLogHandler to simplify customization by subclassing?